pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 dependencies = [
```

Comment:
The github action expects there to be an optional dependency block here, so this also needs reconciling with the template.